### DISCUSSION TITLE
Princes Maker games should be on the compatibility list.

### DISCUSSION THREAD

**Discussion by KoleckOLP** [PCSX2/pcsx2#7720] [Information]
I have tried these games on the PCSX2 v1.7.3807

[SLKA-15031] Princess Maker (Korea)
 - Seems to work just fine

[SLPS-20361] Princess Maker - Refine (Japan)
 - Seems to work just fine
 
[SLPS-20393] Princess Maker 2 (Japan)
 - Boots to a black screen

[SLKA-15033] Princess Maker 2 (Korea)
 - Just like the Japanese one just boot to a black screen

[SLPM-65936] Princess Maker 4 (Japan)
 - Seems to work just fine
 
[SLPM-65935] Princess Maker 4 Limited Edition (Japan)
 - Seems to work just fine

[SLPM-66918] Princess Maker 5 (Japan)
 - Seems to work just fine
 
 My wish is to see these games on the official compatibility list,
 and I also wish there was a way Princess make 2 could be fixed to work.

**Comment by refractionpcsx2** [comment]
You can add them yourself, sign up to the forum, go in to the "Public compatibility list" section and read the sticky threads, they will tell you what to do.

**Reply by KoleckOLP** [reply]
Ok, I will do that hopefully tomorrow.

**Comment by KoleckOLP** [comment]
Here are long s from the two non working games, should I open an issue to get these looked at, and prossibly fixed?

[SLPS-20393] Princess Maker 2 (Japan)
[emulog.txt](https://github.com/PCSX2/pcsx2/files/10316895/emulog.txt)

[SLKA-15033] Princess Maker 2 (Korea)
[emulog.txt](https://github.com/PCSX2/pcsx2/files/10316900/emulog.txt)

**Comment by refractionpcsx2** [comment]
Press F9.

**Comment by refractionpcsx2** [comment]
#7722 will fix it properly for hardware mode.
